### PR TITLE
Add WebListener as an option just like the other samples

### PR DESCRIPTION
**Pull request by avanderhoorn**

### CODE REVIEW

File: samples/RuntimeInfoPageSample/project.json [hunk 1]
```diff
@@ -3,7 +3,11 @@
     "exclude": "wwwroot/**/*.*",
     "dependencies": {
         "Microsoft.AspNet.Diagnostics": "",
-        "Microsoft.AspNet.Server.IIS": "1.0.0-*"
+        "Microsoft.AspNet.Server.IIS": "1.0.0-*",
```
Comment (Tratcher) [inline samples/RuntimeInfoPageSample/project.json]: Remove IIS and replace it with kestrel and platform handler.

File: samples/RuntimeInfoPageSample/project.json [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+        "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
+    },
+    "commands": {
+        "web": "Microsoft.AspNet.Hosting --server Microsoft.AspNet.Server.WebListener --server.urls http://localhost:5000"
```
Comment (Tratcher) [inline samples/RuntimeInfoPageSample/project.json]: the new standard is:
```
"web": "Microsoft.AspNet.Server.Kestrel",
"weblistener": "Microsoft.AspNet.Server.WebListener"
```

**Comment by Eilon** [comment]
@avanderhoorn do you want to update this? Or I can get someone to clean this up.

**Comment by avanderhoorn** [comment]
I'm happy for someone else to take it. Thanks for asking though!

**Comment by Eilon** [comment]
Tracking this as an internal work item instead: https://github.com/aspnet/Coherence-Signed/issues/117

We'll clean up all the repos in one go.

**Pull request closed without merging**